### ISSUE TITLE
TransM does not save embedding.

### ISSUE THREAD

**Issue by YaoXinZhi** [Sujit-O/pykg2vec#115]
When we use the code, we find that the TransM algorithm does not automatically save the embeddings obtained from the training. Is it missing in the code?

**Comment by louisccc** [comment]
Hello Yao, can you paste me the global config printed when the training started?

The embedding should be stored in your dataset folder automatically [dataset folder]/embeddings/transm/*, or if you mean the tensorflow related weights it should be in [dataset folder]/intermediate/*

**Comment by YaoXinZhi** [comment]
Hi, thanks for your reply. This is the setting printed at the beginning of my TransM training. I checked that this is the same configuration as my TransE model, but there is still no TransM embedding under my /data/embedding flder, but there have TransE embedding in /data/embedding/TransE. When I ran several other models, I found that it is not just TransM that does not save the embedding. Because I mainly write Torch, I am not familiar with tensorflow, we are looking for problems, and if they are resolved, I will reply you in time. Thanks again your response.

------------------Global Setting--------------------
                    tmp : ../intermediate
                 result : ../results
                figures : ../figures
    custom_dataset_path : data
         full_test_flag : False
       plot_entity_only : True
              test_step : 10
               test_num : 1000
        disp_triple_num : 20
   log_device_placement : False
           loadFromData : False
             save_model : True
           disp_summary : True
            disp_result : True
         plot_embedding : False
   plot_training_result : True
    plot_testing_result : True
     batch_size_testing : 16
                   hits : [10, 5]
               path_tmp : /store/xyzhou/xzyao/lastest_pykg2vec/data/intermediate
            path_result : /store/xyzhou/xzyao/lastest_pykg2vec/data/results
           path_figures : /store/xyzhou/xzyao/lastest_pykg2vec/data/figures
        path_embeddings : /store/xyzhou/xzyao/lastest_pykg2vec/data/embeddings
---------------------------------------------------

**Comment by louisccc** [comment]
I notice that you are using the old version of the repo, which version or label were you using? Or can you try using the latest version of pykg2vec?

**Comment by YaoXinZhi** [comment]
Thank you for your help. This bug was fixed after I updated the latest version of pykg2vec.
